tests/pgsql/pgsql-7000-ips/README.md

```diff
@@ -0,0 +1,21 @@
+# Description
+
+Showcase engine behavior with simple pgsql stream rules and alert metadata, in
+IPS mode.
+
+## Behavior
+
+The 2 rules will match on the PGSQL queries for `SELECT` (sid 1) and `DELETE FROM`,
+(sid 2). However, the combination of IPS mode matching on the traffic as soon as
+the engine sees it with the continuously available stream buffer leads to way
+more matches than could be initially expected. The `payload_printable` EVE field
+was left to illustrate this.
```

Comment:
to be decided whether we want to have this test to fail, or keep it as is, and add a bug ticket to track this?

Reply:
Why does it fail ?

Reply:
Both for master and with the currently proposed changes we have 37 alerts in IPS mode, which is more than expected - in comparison, we have 8 alerts in IDS mode.
A good illustration of the problem that there is there is because we're getting alerts for the 2 rules (which inspect different contents) for the same packet.
From master to the proposed changes, one of the differences we have that makes the tests fail is the tx_id. 
But my question was more: should we have tests that register what's the expected behavior (the correct number of alerts), versus what we're actually seeing...